initfiles/componentfiles/configschema/xsd/directories.xsd

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+#    HPCC SYSTEMS software Copyright (C) 2018 HPCC Systems®.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+-->
+
+<xs:schema
+        xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified" attributeFormDefault="unqualified"
+        xmlns:hpcc="someuri">
+    <xs:include schemaLocation="types.xsd"/>
+    <hpcc:insert hpcc:schemaPath="/Environment/Software">
+        <!-- is a directories section required in an environment? -->
+        <xs:element name="Directories" hpcc:displayName="Directories" hpcc:itemType="directories" hpcc:class="component">
+            <xs:complexType>
+                <xs:sequence>
+                    <xs:element name="Category" hpcc:displayName="Common" maxOccurs="unbounded" type="directorycategory"
```

Comment:
confirm "Common" is the correct display name for this entry

Reply:
Component owner to confirm this level of correctness. I only translated it from the current XSD.